### PR TITLE
Complete Russian localization

### PR DESCRIPTION
**Pull request by bershan2**
Completed Russian localization with context of each sentence in mind:
0. All items have a translation now;
1. Factually correct, e.g., prior translation had "Включить скрипты" ("Turn scripts on") instead of "Скрипты включены" ("Scripts are on");
2. Proper punctuation, e.g., check-boxes and tool tips terminate with punctuation, while buttons and headings do not;
3. Native Russian sentence structure and word choice instead of English-induced translation;

## Warning:
**In the translators list I included links to GitHub accounts.** See comment in the text box. This should not be a security issue, but I want to inform you about this.

### CODE REVIEW

**Comment by bershan2** [comment]
Unfortunately, comments were separated from the text box, so you'll have to visit [my fork](https://github.com/bershan2/violentmonkey/commit/2b807561ddcae4109e91215420b9ce6e1c32bae4) to see them in text.

File: src/_locales/ru/messages.yml [hunk 1]
```diff
@@ -273,123 +273,125 @@ buttonInstallFromURL:
...
 labelCustomCSS:
   description: Label for custom CSS section.
-  message: ''
+  message: Пользовотельские стили CSS
 descCustomCSS:
   description: Description of custom CSS section.
-  message: ''
+  message: >-
+    Пользовательские стили CSS для страницы Настройки и страницы установки скриптов. 
+	Если Вы не знаете что это такое, пожалуйста не редактируйте этот текст.
```
Comment (gera2ld) [inline src/_locales/ru/messages.yml]: Here is a syntax error. You may put them in one line if indentation is too much trouble.

Comment (bershan2) [inline src/_locales/ru/messages.yml]: Sorry, did not notice.

**Comment by bershan2** [comment]
Let me merge these commits.

**Comment by bershan2** [comment]
How does it look now?

**Comment by bershan2** [comment]
Cool!

**Comment by bershan2** [comment]
Do you have a forum for Violentmonkey? I have a few general questions about implementation, however I do not want to pollute your issues. Or should I write you an email to i@gerald.top? Or is it gera2ld@163.com?

**Comment by gera2ld** [comment]
No forum yet. Both emails are OK, the former preferred.

**Pull request merged**